src/components/HeaderPageLayout.tsx

```diff
@@ -69,7 +73,7 @@ function HeaderPageLayout({
             shouldEnablePickerAvoiding={false}
             includeSafeAreaPaddingBottom={false}
             offlineIndicatorStyle={[appBGColor]}
-            testID={HeaderPageLayout.displayName}
+            testID={testID || HeaderPageLayout.displayName}
```

Comment:
And with this being a required param, we do not need a fallback:

```suggestion
            testID={testID}
```